app/assets/javascripts/discourse/app/templates/components/pwa-install-banner.hbs

```diff
@@ -4,12 +4,13 @@
       <span>
         {{discourse-linked-text
           action=(action "turnOn")
-          translatedText=(i18n "pwa.install_banner" title=siteSettings.title)
+          text="pwa.install_banner"
+          textParams=(hash title=siteSettings.title)
```

Comment:
doesn't look related to actions

Reply:
my bad, while fixing the actions I found this overwritten CP so I fixed that too. I'll split it out.